test/jdk/java/awt/Frame/GetBoundsResizeTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 1998, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Button;
+import java.awt.Frame;
+import java.awt.EventQueue;
+import java.awt.Robot;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+
+/*
+ * @test
+ * @bug 4103095
+ * @summary Test for getBounds() after a Frame resize.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual GetBoundsResizeTest
+*/
+
+public class GetBoundsResizeTest {
+    private static final String INSTRUCTIONS = """
+            0. There is a test window with a "Press" button,
+                Its original bounds should be printed in the text area below.
+            1. Resize the test window using the upper left corner.
```

Comment:
I found it impossible to resize the window by top left corner on Windows… The frame is so small that the Minimize button in the title bar covers the icon.

Reply:
Hmm, works for me:

[Screencast.webm](https://github.com/openjdk/jdk/assets/77687766/8d6d5edd-178c-4dd4-9385-9492fde3ce1a)

Anyway, I changed the default size of the window.

Reply:
I'm using Windows 11 where the window borders are rounded, it could be the reason why it didn't work for me: the target area to grab the border is already covered by the _Minimise_ button.